src/main/java/com/bjmbjm/code401d56day12/firstSpringDemo/HelloWorldController.java

```diff
@@ -18,4 +24,20 @@ public String getHelloWorld() {
     public String getHelloToUser(@PathVariable String name, @PathVariable int from) {
         return "Hello, " + name + " from Agent " + from * 25;
     }
+    @GetMapping("/capitalize/{str}")
+    public String capitalizeStr(@PathVariable String str) {
+
+        if(str.equals("")){
+            throw new IllegalArgumentException("Path variable cannot be a empty string.");
+        }
+        return str.toUpperCase();
+    }
+    @GetMapping("/reverse")
```

Comment:
This route doesn't have requestParams so it's unable to accept anything form the user